Components/Alert.html

```diff
@@ -0,0 +1,221 @@
+        </div>
+    </div>
+
+    <!---Hamburger List -->
+    <div class="hamburger">
+        <div class="hamburger__header--wrapper">
+            <h1 class="hamburger__title">AQUA_UI</h1>
+            <img class="hamburger__close-btn" src="/assets/images/Hamburger Close.png" alt="Hamburger">
+        </div>
+        <ul class="hamburger__list">
+            <a href="/index.html">
+                <li class="hamburger__item">Home</li>
+            </a>
+            <a href="/Components/Avatar.html">
+                <li class="hamburger__item">Avatar</li>
+            </a>
+            <a href="/Components/Alert.html">
+                <li class="hamburger__item">Alert</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Badge</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Button</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Image</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Card</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Input</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Modal</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Navigation</li>
+            </a>
+            <a href="">
+                <li class="hamburger__item">Toast</li>
+            </a>
+        </ul>
+    </div>
+    <main>
+        <!-- sidebar -->
+        <div class="sidebar">
+            <h1 class="sidebar__title">Components</h1>
+            <ul class="sidebar__list">
+                <a href="/Components/Avatar.html">
+                    <li class="sidebar__item">Avatar</li>
+                </a>
+                <a href="/Components/Alert.html">
+                    <li class="sidebar__item">Alert</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Badge</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Button</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Image</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Card</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Input</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Modal</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Navigation</li>
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Toast</li>
+                </a>
+            </ul>
+        </div>
+
+        <!-- Alert -->
+        <!-- Alert Intro -->
+        <div class="alert card">
```

Comment:
you will be creating a card component so is it good to use a class name as card over here?